docs/en/observability/elastic-entity-model.asciidoc

```diff
@@ -0,0 +1,67 @@
+[[elastic-entity-model]]
+= Elastic Entity Model
+
+preview::[]
+
+The Elastic Entity Model consists of:
+
+* a data model and related entity indices
+* an Entity Discovery Framework, which consists of {ref}/transforms.html[transforms] and {ref}/ingest.html[Ingest pipelines] that read from signal indices and write data to entity indices
+* a set of management APIs that empower entity-centric Elastic solution features and workflows
+
+In the context of Elastic Observability,
+an _entity_ is an object of interest that can be associated with produced telemetry and identified as unique.
+Note that this definition is intentionally closely aligned to the work of the https://github.com/open-telemetry/oteps/blob/main/text/entities/0256-entities-data-model.md#data-model[OpenTelemetry Entities SIG].
+Examples of entities include (but are not limited to) services, hosts, and containers.
+
+The concept of an entity is important as a means to unify observability signals based on the underlying entity that the signals describe.
+
+[NOTE]
+====
+* The Elastic Entity Model currently supports the <<new-experience-services,new service inventory experience>> limited to service-based entities (as identified by `service.name`) located in data identified by `logs-*` and `trace-*` index patterns
+* During Technical Preview, Entity Discovery Framework components are not enabled by default
+====
+
+[discrete]
+== Enable the Elastic Entity Model
+
+During Technical Preview,
+the Elastic Entity Model is enabled when you turn on the entity-centric service inventory described in <<new-experience-services>>.
+
+The following {es} privileges are required:
+
+[cols="1,1"]
+|====
+|**Index privileges**
+|names: [`.entities*`], privileges: [`create_index`, `index`, `create_doc`, `auto_configure`, `read`]
+
+names: [`logs-*`, `filebeat*`, `metrics-*`, `metricbeat*`, `traces-*`, `.entities*`], privileges: [`read`, `view_index_metadata`],
+|**Cluster privileges**
+|`manage_transform`, `manage_ingest_pipelines`, `manage_index_templates`
+
+|**Application privileges**
+|application: `kibana-.kibana`, privileges: [`saved_object:entity-definition/*`, `saved_object:entity-discovery-api-key/*`], resources: [*]
+|====
+
+For more information, refer to {ref}/security-privileges.html[Security privileges] in the {es} documentation.
+
+[discrete]
+== Disable the Elastic Entity Model
+
+From the Dev Console, run the command: `DELETE kbn:/internal/entities/managed/enablement`
+
+The following {es} privileges are required to delete {es} resources:
+
+[cols="1,1"]
+|====
+|**Index privileges**
+|names: [`.entities*`], privileges: [`delete_index`]
+
+names: [`logs-*`, `filebeat*`, `metrics-*`, `metricbeat*`, `traces-*`, `.entities*`], privileges: [`read`, `view_index_metadata`],
```

Comment:
this is not required to disable eem

Reply:
Sorry, copy/paste error. I should have double-checked.